requirements.txt

```diff
@@ -1,3 +1,4 @@
+black==21.6b0
 packaging==20.3
 pylint
```

Comment:
Should this be pinned as well?

Reply:
> I noticed you pinned Black, and I wasn't sure if we should be pinning Pylint as well.

I didn't pin it since the locally installed module is used to verify that a library is passing with the latest pylint, in `lib/circuitpython_library_validators.py::validate_passing_linting`.

However, in looking back at changes to libs/cookiecutter, it seems they're all now pinned to 2.7.1  via pre-commit-config. Which brings the question: is `validate_passing_linting` overachieving by linting with the latest version?